src/components/ui/rewardsCalculatorDialog.js

```diff
@@ -60,7 +65,10 @@ const RewardsCalculatorDialog = ({
 
       const aprAcrossLockPeriod = userRewardsPlusInitialLock / inputIQ;
 
-      setAprDividedByLockPeriod((aprAcrossLockPeriod / years) * 100);
+      let percentage = 100;
+      if (years < 1) percentage = (years * 100) / 1;
```

Comment:
why are you dividing by 1?

Reply:
it is because you need to divide it by 1 year to get the proportional percentage of the input period, like the rule of 3

Reply:
Any number divided by 1 equals itself